Gemfile

```diff
@@ -20,27 +18,28 @@ gem 'local_time', '~> 2.0'
 gem 'pg', '~> 0.18'
 gem 'puma', '~> 3.12'
 gem 'rack-cors', '~> 1.0', '>= 1.0.2'
-gem 'rails', '~> 5.1.6'
+gem 'rails', '~> 5.2.0'
 gem 'sass-rails', '~> 5.0'
 gem 'slack-notifier', '~> 2.3.2'
 gem 'turbolinks', '~> 5'
 gem 'uglifier', '>= 1.3.0'
 gem 'webpacker', '~> 3.5'
 
 group :test do
-  gem 'minitest-rails-capybara'
+  gem 'capybara', '~> 2.18'
```

Comment:
This capybara gem is not required when using the `minitest-rails-capybara` gem.

Reply:
This is technically correct, but because the gemspec for minitest-rails-capybara includes only capybara 2.7, and System Tests require greater than that, we can manually force a greater version of capybara by explicitly stating it.

Reply:
That makes sense.

Reply:
Just a heads up, it looks like `minitest-rails-capybara` is no longer being maintained since the system tests were added to rails. Here is a blog from the maintainer.

https://github.com/blowmage/minitest-rails-capybara/issues/45